### PR TITLE
[WIP]Changes to Server docs for v2.18

### PR DESCRIPTION
**Pull request by rosieyohannan**
Various changes to the update guide inc:

* Previously the update guide only talked through the process as far as updating replicated, this PR adds a section explaining the releases page in replicated, plus install buttons, how long the process takes etc.

### CODE REVIEW

File: jekyll/_cci2/monitoring.adoc [hunk 1]
```diff
@@ -576,6 +576,95 @@ Custom Metrics using a Telegraf configuration file may be configured in addition
 .Custom Metrics
 image::metrics-custom.png[Custom]
 
+=== Available Metrics
```
Comment (rosieyohannan) [inline jekyll/_cci2/monitoring.adoc]: @GERey custom metrics info here for review and completion when you have all the metrics descriptions

File: jekyll/_cci2/updating-server.adoc [hunk 2]
```diff
@@ -5,44 +5,187 @@
...
 
-* Your installation is Ubuntu 14.04 or 16.04 based.
-* You are running replicated version 2.10.3<= on your services machine. Check by running the following:
+* Your installation is Ubuntu 14.04 or 16.04 based
+* You are running replicated version >=2.10.3 on your services machine. Check by running the following:
 +
 ```shell
 replicated --version
 ```
+// do they need to SSH into the services machine and run this? Actually we need to point them to what to do if their version of Replicated is pre 2.10.3 here
```
Comment (rosieyohannan) [inline jekyll/_cci2/updating-server.adoc]: @GERey there's a few questions for you in comments around here :-)

Comment (GERey) [reply jekyll/_cci2/updating-server.adoc]: Mostly they need to do this. There is a way for people to possibly upgrade using replicated itself, but I trust the ssh process more than the replicated auto process. A requirement for installing v2 of server is to go to replicated 2.10.3. Nobody on 2.0 should be using a version of replicated before 2.10.3. At least I hope so ;)

File: jekyll/_cci2/updating-server.adoc [hunk 3]
```diff
@@ -53,6 +196,7 @@ replicatedctl app stop
 ```
 +
 Application shutdown takes a few minutes. Please check the administration dashboard and wait for the status to become “Stopped” before continuing. You can also run the following command to view the app status:
+// is this what we want them to do generally? In the step above we're saying to stop replicated from the dashboard
```
Comment (GERey) [inline jekyll/_cci2/updating-server.adoc]: the replicated command above is really the equivalent of pressing the `stop now` button in the UI. It doesn't stop replicated but only stops CircleCI. If we wanted to stop replicated we'd do `sudo service replicated stop` or something like that.

File: jekyll/_cci2/updating-server.adoc [hunk 4]
```diff
@@ -76,6 +220,7 @@ Example Output:
 ```
 
 . For the replicated update to succeed, it is necessary to update docker to the recommended version, 17.12.1:
+// should there be a step to check Docker version first?
```
Comment (GERey) [inline jekyll/_cci2/updating-server.adoc]: ah that's probably a good thing to do before starting this whole process. They can do a version check by running `docker version` .

File: jekyll/_cci2/updating-server.adoc (hunk 2 above)
Comment (rosieyohannan) [reply jekyll/_cci2/updating-server.adoc]: Done

File: jekyll/_cci2/updating-server.adoc [hunk 5]
```diff
@@ -130,5 +280,15 @@ Example output:
     }
 ]
 ```
+
+=== 3. Upgrade CircleCI Server Version
```
Comment (rosieyohannan) [inline jekyll/_cci2/updating-server.adoc]: @jkzilla here's the upgrade section, if you have any comments.

File: jekyll/_cci2/updating-server.adoc (hunk 4 above)
Comment (rosieyohannan) [reply jekyll/_cci2/updating-server.adoc]: Done

File: jekyll/_cci2/updating-server.adoc [hunk 6]
```diff
@@ -1,58 +1,207 @@
...
-. Take a snapshot of your installation so you can rollback later if necessary (optional but recommended) - steps below
-. Check you are running Docker v17.12.1 and update if necessary
-. Update Replicated to install the new version of CircleCI Server - steps below
+WARNING: Before upgrading please read and follow the steps below if you have **ever had issues with renaming an organization within CircleCI**
+or you suspect that an **organization rename might have happened at any point**.
 
-== Snapshot for Rollback
+. SSH into your Services machine
+. REPL into `workflows-conductor` by running the following: `sudo docker exec -it workflows-conductor lein repl :connect 6005`
+. Copy/paste this script into the REPL session. It will run migration and output current progress.
```
Comment (rosieyohannan) [inline jekyll/_cci2/updating-server.adoc]: @GERey org rename script here :-)

File: jekyll/_cci2/customizations.adoc [hunk 7]
```diff
@@ -134,3 +134,80 @@ export CIRCLE__OUTER__LOGIN_BANNER_MESSAGE="<insert-your-message-here>
...
+```
+. Add your required customizations to the file, then save and exit vim with `:wq` - see below for options and formatting.
++
+WARNING: It is important to use the IDs listed in the example below for both `machine` and Docker resources classes. 
+. Run:
++
+```shell
+echo 'export CIRCLE_DISPATCHER_RESOURCE_DEF=/circleconfig/picard-dispatcher/resource-definitions.edn' | sudo tee /etc/circleconfig/picard-dispatcher/customizations
+```
+. Restart the CircleCI Server application ADD IN HOW HERE!
```
Comment (rosieyohannan) [inline jekyll/_cci2/customizations.adoc]:
```suggestion
. Restart the CircleCI Server application. The application can be stopped and started again from the Management Console Dashboard (for example, `<circleci-hostname>.com:8800`).
```

File: jekyll/_cci2/customizations.adoc [hunk 8]
```diff
@@ -134,3 +134,80 @@ export CIRCLE__OUTER__LOGIN_BANNER_MESSAGE="<insert-your-message-here>
 
 .Login Screen Banner Example
 image::banner.png[]
+
+=== Resource Classes
```
Comment (rosieyohannan) [inline jekyll/_cci2/customizations.adoc]: Customize resource classes

File: jekyll/_cci2/vm-service.adoc [hunk 9]
```diff
@@ -42,9 +44,14 @@ You can provide a custom https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMI
 * Us-west-1:ami-06f6efb13d9ccf93d
 * Us-west-2:ami-0b5b8ad02f405a909
 
-== Customization
+== Customizing VM Service Images
```
Comment (rosieyohannan) [inline jekyll/_cci2/vm-service.adoc]: VM Service images

File: jekyll/_cci2/updating-server.adoc (hunk 6 above)
Comment (GERey) [reply jekyll/_cci2/updating-server.adoc]: I think the script as it should be exists below so I'll resolve this :) Thanks!

**Pull request merged**